services/shhext/chat/sql_lite_persistence.go

```diff
@@ -51,6 +53,63 @@ func NewSQLLitePersistence(path string, key string) (*SQLLitePersistence, error)
 	return s, nil
 }
 
+func MigrateDBFile(oldPath string, newPath string, key string) error {
+	_, err := os.Stat(oldPath)
+
+	// No files, nothing to do
+	if os.IsNotExist(err) {
+		return nil
+	}
+
+	// Any other error, throws
+	if err != nil {
+		return err
+	}
+
+	if err = os.Rename(oldPath, newPath); err != nil {
```

Comment:
You can use `err := os.Rename` here. It's suspicious and makes me thinking "why `err =` instead of `err :=` :D